tests/common/dualtor/control_plane_utils.py

```diff
@@ -106,8 +112,8 @@ def _dump_db(self, db, key_pattern):
 
     def verify_db(self, db):
         pytest_assert(
-            wait_until(self.VERIFY_DB_TIMEOUT, 10, 0,
-                       self.get_mismatched_ports, db),
+            wait_until_with_last_check(self.VERIFY_DB_TIMEOUT, 10, 0,
+                                       self.get_mismatched_ports, db),
```

Comment:
Did you use tab here?

Reply:
yes, and those two lines are auto-aligned.